lib/index.js

```diff
@@ -52,6 +52,28 @@ internals.isFunction = function(functionToCheck) {
   );
 };
 
+internals.getKeys = async function(decoded, options) {
+  // if keyFunc is function allow dynamic key lookup: https://git.io/vXjvY
+  const { key, ...extraInfo } = internals.isFunction(options.key)
+    ? await options.key(decoded)
+    : { key: options.key };
+  const keys = Array.isArray(key) ? key : [key];
+  return { keys, extraInfo };
+};
+
+internals.verifyJwt = function(token, keys, options) {
+  let error;
+  let verifyOptions = options.verifyOptions || {};
+  for (const k of keys) {
+    try {
+      return JWT.verify(token, k, verifyOptions);
```

Comment:
I think `verifyOptions` is ok to be undefined:

```js
internals.verifyJwt = function(token, keys, options) {
	let error;
	for (const k of keys) {
		try {
			return JWT.verify(token, k, options.verifyOptions);
		} catch (verify_err) {
			error = verify_err;
		}
	}
	throw error;
};
```

Reply:
Hmmm. It was like this before I got here :D @nelsonic do you have an opinion?

Reply:
_Agree_ that it's "OK" for the `verifyOptions` to be `undefined`, 
I just _prefer_ a default to `undefined` because `undefined` is lame hence the expression.
But if all we are doing is passing the the `options.verifyOptions` directly into the `JWT.verify` method then there's no need for the `let verifyOptions = ...`. 

Conclusion: streamline (_remove_) it if you still have your editor open. 👍

Reply:
https://github.com/dwyl/hapi-auth-jwt2/pull/287/commits/68f610245b0b7d001e9c8672b100b31e2b332302